cmd/src/api.go

```diff
@@ -65,7 +66,7 @@ Examples:
 		if query == "" {
 			// Read query from stdin instead.
 			if isatty.IsTerminal(os.Stdin.Fd()) {
-				return &usageError{errors.New("expected query to be piped into 'src api' or -query flag to be specified")}
+				return cmderrors.Usage("expected query to be piped into 'src api' or -query flag to be specified")
```

Comment:
I pulled these errors out into a separate package because one or two functions in the `ui` package that print them needed to check for their type.